rust/noosphere-cli/src/native/helpers/workspace.rs

```diff
@@ -105,9 +113,54 @@ pub async fn start_name_system_server(ipfs_url: &Url) -> Result<(Url, JoinHandle
 pub struct SphereData {
     /// The identity of the sphere
     pub identity: Did,
+    /// The recovery mnemonic of the sphere
+    pub mnemonic: Mnemonic,
     /// The temporary [Workspace] of the sphere
     pub workspace: Workspace,
-    _temp_dirs: (tempfile::TempDir, tempfile::TempDir),
+    /// The name of the device key in use when the sphere was created
+    pub key_name: String,
+    temp_dirs: (tempfile::TempDir, tempfile::TempDir),
+}
+
+impl SphereData {
+    pub async fn simulate_storage_corruption(&self) -> Result<()> {
```

Comment:
very nice

Reply:
I'm a little surprised to not see this behind a feature flag like `test` or maybe in a different module. Do we think users of noosphere-cli will do storage corruption simulation?

Reply:
It is in fact behind a feature flag called `helpers`, which is our conventional name for modules that contain test helpers.

Reply:
See: https://github.com/subconsciousnetwork/noosphere/blob/74b24c842150eeb1d5c035f545a4090d6786ed36/rust/noosphere-cli/src/native/mod.rs#L11-L12